go.mod

```diff
@@ -6,29 +6,35 @@ require (
 	github.com/Flaque/filet v0.0.0-20201012163910-45f684403088
 	github.com/Jeffail/gabs/v2 v2.6.0
 	github.com/cenkalti/backoff/v4 v4.1.0
-	github.com/cucumber/godog v0.11.0
-	github.com/cucumber/messages-go/v10 v10.0.3
+	github.com/cucumber/godog v0.11.0 // indirect
```

Comment:
I think this is going to conflict with #1588

Reply:
Let's merge #1588 first then ill fix the conflicts